utils/display.go

```diff
@@ -68,6 +68,52 @@ func PrintEndpointsTable(ends []meroxa.Endpoint) {
 	fmt.Println(EndpointsTable(ends))
 }
 
+func header(h string) string {
+	header := h + "\n"
+	for range header {
+		header += "="
+	}
+	header += "\n"
+
+	return header
+}
+
+func ResourceTable(res *meroxa.Resource) string {
+	mainHeader := header(fmt.Sprintf("Resource %s", res.Name))
```

Comment:
I'm not sure I'm in love with this header. It seems kinda redundant if you look at it right next to the command you just executed:

```
❯ .m resource describe resource
Resource resource
==================
    ID:   444
  NAME:   resource
  TYPE:   postgres
   URL:   url
 Status   starting
```

Maybe we should get rid of it.

Reply:
Header was removed in https://github.com/meroxa/cli/pull/148/commits/051d7024e7ebcfe6ac3b9be235657a1f37c3b45d